ground/openpilotgcs/src/plugins/config/cfg_vehicletypes/configmultirotorwidget.cpp

```diff
@@ -787,7 +787,7 @@ QString ConfigMultiRotorWidget::updateConfigObjectsFromWidgets()
 
             // Tricopter : Yaw mix slider value applies to servo (was fixed)
             // Get absolute MixerValueYaw, no servo reverse when Reverse All Motors is checked
-            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(getMixerValue(mixer, "MixerValueYaw")) * 1.27);
+            setMixerVectorValue(mixer, channel, VehicleConfig::MIXERVECTOR_YAW, abs(int(getMixerValue(mixer, "MixerValueYaw"))) * 1.27);
```

Comment:
I've verified that both abs and fabs give the same results with:
```
#include <math.h>
#include <stdlib.h>
#include <assert.h>

int main() {
  for (int i = -100; i <= 100; i++) {
	assert(abs(double(i)) * 1.27 == fabs(double(i)) * 1.27);
  }

  return 0;
}
```
```
g++ f.cpp -Wno-absolute-value
```
```
$ ./a.out
```
So we could probably change to fabs.